docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md

```diff
@@ -21,36 +21,49 @@ git checkout v1.1
 
 2. **Install prerequisites**. Make sure you have installed all the tools specified in the prerequisites https://magma.github.io/magma/docs/basics/prerequisites#prerequisites
 
-3. **Build and create deb package**. In your local magma repo, go to the path `magma/lte/gateway` and run the command `fab dev package:vcs=git`
+3. **Build and create deb package**.
+    To build an AGW package, use the script located at `$MAGMA_ROOT/lte/gateway/fabfile.py`. The commands below will create a vagrant machine, then build and create a deb package.
+
+    The following commands are to be run from `$MAGMA_ROOT/lte/gateway` on your host machine.
+    To create a package for production, run
+
+    ```bash
+    fab test package:vcs=git
```

Comment:
This is... weird that we're using "test" to create a pkg for production. Seems like a clear antipattern. But for a followup task I suppose

Reply:
Ya have a PR to rename it to 'release'